cas-airflow/values.yaml

```diff
@@ -2,10 +2,10 @@ airflow:
   airflow:
     image:
       repository: docker-registry.default.svc:5000/wksv3k-tools/cas-airflow
-      tag: 0.0.1
+      tag: '0.1.0-pre.1.0'
```

Comment:
while this does fit the [grammar of a semver tag](https://semver.org/#backusnaur-form-grammar-for-valid-semver-versions) maybe we could use build identifiers for pre-releases? See [spec item 10](https://semver.org/#spec-item-10) of the spec.